vulcan/_data.py

```diff
@@ -56,6 +57,18 @@ async def get_addressbook(
         """
         return Addressbook.get(self._api, **kwargs)
 
+    async def get_messages(
+        self, last_sync: datetime = None, folder=1, **kwargs
+    ) -> Union[AsyncIterator[Message], List[int]]:
+        """Yields messages received by the currently registered account.
+
+        :param `datetime.datetime` last_sync: date of the last sync,
+            gets only the objects updated since this date
+        :param int folder: message folder
```

Comment:
Also, I didn't notice this before, but it might be helpful to tell which message folder has which number here.

Reply:
Unfortunately, I don't know what folder number 3 is, and I didn't want to write something I don't know.

Reply:
Number 3 would be a deleted message, i.e. the "trash" folder.